### PR TITLE
website: update select documentation & demo

### PR DESCRIPTION
**Pull request by idakukimiya**
- updated documenation
- added more live example

### CODE REVIEW

**Review by elephantcatdog — COMMENTED** [review]
Don't forget to remove the placeholder!

File: apps/website/package.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
     "@astrojs/sitemap": "^1.0.1",
     "@fontsource/noto-sans": "^4.5.11",
     "@fontsource/noto-sans-mono": "^4.5.11",
-    "@itwin/itwinui-react": "*",
```
[the remark this answers is not in the document]
Comment (idakukimiya) [reply apps/website/package.json:19]: I am not sure what happened there. Do you know what it was before, I can it change it back?

File: apps/website/src/examples/Select.disable.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        value: 3,
+        label: 'Item #3',
+      },
+    ],
+    placeholder = 'Placeholder text',
+    ...rest
+  } = args;
+  const [value, setValue] = useState<number | undefined>(undefined);
+  return (
+    <div
```
Comment (gretanausedaite) [inline apps/website/src/examples/Select.disable.tsx:29]: Remove this additional div

File: apps/website/src/pages/docs/select.mdx [hunk 3]
```diff
@@ -22,7 +22,53 @@ import * as AllExamples from '~/examples';
...
+  <AllExamples.SelectIconExample client:load />
+</LiveExample>
+
+### Disable
+
+<LiveExample src='Select.disable.tsx'>
+  <AllExamples.SelectDisableExample client:load />
+</LiveExample>
+
+The menu can be disabled as its default state, without any option selected; it can be disaled with an option selected; and options in the dropdown menu can be disabled so the user cannot select one or more specific options, depending on context.
```
Comment (gretanausedaite) [inline apps/website/src/pages/docs/select.mdx:39]: Check spelling.
This paragraph has "disaled" instead of "disabled"

**Review by gretanausedaite — COMMENTED** [review]
Are you running the app when making changes? It might help you visualise how users perceive this text.

File: apps/website/package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (elephantcatdog) [reply apps/website/package.json:19]: image
If you go to the 'Files changed' tab you can see what the code used to look like.

You'll need to change it back and run `yarn` again to run the website again on your local branch.

You will also need to undo the changes to the `yarn.lock` file. You can go to the `Files changed` tab and find the 
image

**Review by gretanausedaite — APPROVED** [review]

**Pull request merged**